hebrew/shame-on-you/boring/you-failed.md

```diff
@@ -0,0 +1,2 @@
+מצטער, אך נראה כאילו נכשלת בהרפתקאה שלך...
+[נסה שוב!](https://github.com/Urlord2089/create-your-own-adventure/blob/appropriate_hebrew/hebrew/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.